lib/plan-builder-generated.js

```diff
@@ -19,6 +19,7 @@
 
 const types    = require('./server-types-generated.js');
 const bldrbase = require('./plan-builder-base.js');
+let patchArgs = [];
```

Comment:
I'm wondering if there's a thread safety issue here. If the user has a middle tier that uses the Node Client and two separate threads are building a plan, do they get their own instance of patchArgs, or are they trying to use the same instance of patchArgs? If it's the latter, they'd run into a thread safety issue as both threads would be populating this array, likely causing both plans to break.